cc_plugin_imos/imos.py

```diff
@@ -1262,3 +1257,42 @@ def check_quality_control_global(self, dataset):
             )
 
         return ret_val
+
+    def check_acknowledgement(self, dataset):
+        """
+        Check the global acknowledgement attribute and ensure it contains the
+        required text.
+        """
+        ret_val = []
+
+        old_pattern = (r"Any users( \(including re-?packagers\))? of IMOS data( \(including re-?packagers\))? "
+                       r"are required to clearly acknowledge the source of the material( derived from IMOS)? "
+                       r"in (this|the) format: \"Data was sourced from the Integrated Marine Observing System "
+                       r"\(IMOS\) - IMOS is( a national collaborative research infrastructure,)? supported by "
+                       r"the Australian Government")
+        new_pattern = (r"Any users of IMOS data are required to clearly acknowledge the source "
```

Comment:
rather than using old, new, I would add either a date or a version number as a suffix, just in case we have to change this again in the future

Reply:
Yeah, good idea. I thought of that in updating the unittest, but forgot about it here. I'll update.

Reply:
Done!